models/classes/runner/features/TestRunnerFeature.php

```diff
@@ -150,11 +156,28 @@ public function isEnabledByDefault()
         return $this->isEnabledByDefault;
     }
 
+    /**
+     * Is feature activated
+     * @return boolean
+     */
+    public function isActive()
+    {
+        return $this->active;
+    }
+
+    /**
+     * @param boolean $active
+     */
+    public function setActive($active)
+    {
+        $this->active = $active;
+    }
+
     /**
      * User-friendly localized label for the feature
      * @return string
      */
-    abstract function getLabel();
+    abstract public function getLabel();
```

Comment:
You can remove getLabel and getDescription abstract method, because it is already constraint in interface